src/components/SyncServiceSignIn/index.js

```diff
@@ -113,15 +112,10 @@ function GitLab() {
   const defaultProject = 'https://gitlab.com/your/project';
   const [project, setProject] = useState(defaultProject);
   const handleSubmit = (evt) => {
```

Comment:
Minor issue here: `yarn eslint` will complain that `evt` is never used. I'm not removing it as long as the validation of the `projectId` is still in question.